src/Plugin/Field/FieldWidget/EnhancedButtonLinkWidget.php

```diff
@@ -31,6 +31,7 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
       '#default_value' => isset($items[$delta]->options['style']) ? $items[$delta]->options['style'] : '',
       '#description' => $this->t('Select the style of the button.'),
       '#options' => [
+        'default' => $this->t('default'),
```

Comment:
Please update 'default' to 'Default', so it should be same as:
`$this->t('Default'),`